app/controllers/application_controller/feature_flags_dependency.rb

```diff
@@ -28,4 +28,9 @@ def lti_launch_enabled?
     GitHubClassroom.flipper[:lti_launch].enabled?
   end
   helper_method :lti_launch_enabled?
+
+  def github_template_repos_enabled?
```

Comment:
How does this work? It looks like in the code, we're checking if template repos are enabled on an organization, not a user. We should probably always check on orgs and not users.

Reply:
Good catch!